ebos/eclmpiserializer.hh

```diff
@@ -498,6 +524,20 @@ protected:
           variadic_call(std::forward<Args>(args)...);
     }
 
+    template<std::size_t I = 0, typename Tuple>
+    typename std::enable_if<I == std::tuple_size<Tuple>::value, void>::type
```

Comment:
Now I wonder why we don't treat pairs like tuple. :laughing: They should be accessible just like these.

Reply:
hmm, that's a good point. i'll see if i can make that work.